IBM Cloud/WML/notebooks/watsonx/Prompt Evaluation with watsonxgov.ipynb

```diff
@@ -157,7 +182,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In order to set up a development type subscription, the PTA must be within the project. Please supply the project ID where the PTA needs to be created."
+    "In order to set up a development type subscription, the PTA must be within the project. Please supply the project ID where the PTA needs to be created. Your OpenScale instance must be associated with this project"
```

Comment:
`Your OpenScale instance must be associated with this project` is untrue. We associate WML instant and not OpenScale instance